features/zoek-met-bsn.feature

```diff
@@ -0,0 +1,83 @@
+        | burgerservicenummer |
+        | 999999321           |
+        Als personen wordt gezocht met de volgende parameters
+        | naam                | waarde                     |
+        | type                | ZoekMetBurgerservicenummer |
+        | burgerservicenummer | 999999321                  |
+        | fields              | burgerservicenummer        |
+        Dan bevat de response alleen personen met de volgende gegevens 
+        | burgerservicenummer |
+        | 999999321           |
+
+    Scenario: Zoek met meerdere valide burgerservicenummers
+
+        Gegeven het systeem heeft een persoon met de volgende gegevens
+        | burgerservicenummer |
+        | 999999321           |
+        En het systeem heeft een persoon met de volgende gegevens
+        | burgerservicenummer |
+        | 999995492           |
+        En het systeem heeft een persoon met de volgende gegevens
+        | burgerservicenummer |
+        | 999991802           |
+        Als personen wordt gezocht met de volgende parameters
+        | naam                | waarde                        |
+        | type                | ZoekMetBurgerservicenummer    |
+        | burgerservicenummer | 999999321,999995492,999991802 |
+        | fields              | burgerservicenummer           |
+        Dan bevat de response alleen personen met de volgende gegevens 
+        | burgerservicenummer |
+        | 999999321           |
+        | 999995492           |
+        | 999991802           |
+
+    @fout-case
+    Scenario: Zoek zonder burgerservicenummers
+        Als personen wordt gezocht met de volgende parameters
+        | naam   | waarde                     |
+        | type   | ZoekMetBurgerservicenummer |
+        | fields | burgerservicenummer        |
+        Dan bevat de response een problemDetails met de volgende velden
+        | naam   | waarde                                        |
+        | title  | Één of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+
+    @fout-case
+    Abstract Scenario: <titel>
+        Als personen wordt gezocht met de volgende parameters
+        | naam                | waarde                     |
+        | type                | ZoekMetBurgerservicenummer |
+        | burgerservicenummer | <burgerservicenummers>     |
+        | fields              | burgerservicenummer        |
+        Dan bevat de response een problemDetails met de volgende velden
+        | naam   | waarde                                        |
+        | title  | Één of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+
+        Voorbeelden:
+        | burgerservicenummers | titel                                   |
+        |                      | Zoek met leeg burgerservicenummer lijst |
+        | 1234567890           | Zoek met ongeldig burgerservicenummer   |
```

Comment:
ook hier is het denk ik goed om te laten zien hoe de gebruiker informatie (details) krijgt over de fout. "Een of meerdere parameters zijn niet correct." heel erg generiek en weinig behulpzaam. Als gebruiker weet ik nu nog niet hoe of waar ik de werkelijke oorzaak van de fout kan vinden (in dit geval burgerservicenummer voldoet niet aan pattern)

Reply:
Sorry dat het niet duidelijk is dat dit is weggelaten omdat ik tijdens het schrijven van de feature en het implementeren van de bijbehorende automation code niet in de foutafhandeling feature wil gaan zoeken welke fout waarden er allemaal moet worden ingevuld.
Zo te lezen heb ik de prioriteiten verkeerd ingeschat en heb ik mijn tijd eerst moeten besteden aan het correct en volledig maken van de scenario's.